5e-SRD-Monsters.json

```diff
@@ -204,7 +204,57 @@
     "special_abilities": [
       {
         "name": "Spellcasting",
-        "desc": "The acolyte is a 1st-level spellcaster. Its spellcasting ability is Wisdom (spell save DC 12, +4 to hit with spell attacks). The acolyte has following cleric spells prepared:\n\n- Cantrips (at will): light, sacred flame, thaumaturgy\n- 1st level (3 slots): bless, cure wounds, sanctuary"
+        "desc": "The acolyte is a 1st-level spellcaster. Its spellcasting ability is Wisdom (spell save DC 12, +4 to hit with spell attacks). The acolyte has following cleric spells prepared:\n\n- Cantrips (at will): light, sacred flame, thaumaturgy\n- 1st level (3 slots): bless, cure wounds, sanctuary",
+        "spellcasting": {
+          "level": 1,
+          "ability": {
+            "name": "WIS",
+            "url": "http://www.dnd5eapi.co/api/ability-scores/5"
+          },
+          "dc": 12,
+          "modifier": 4,
+          "components_required": [
+            "V",
+            "S",
+            "M"
+          ],
```

Comment:
Some component_required sections are on one line, others are on several lines. Consistency should be the key

Reply:
Silly auto styler. I'll see if I can fix it.

Reply:
@ogregoire  Fixed!